plugins/telemetry/telemetry.h

```diff
@@ -250,6 +273,14 @@ class Telemetry : public PluginBase {
      */
     Result set_rate_rc_status(double rate_hz);
 
+    /**
+     * @brief Set rate of position velocity local updates (asynchronous).
```

Comment:
"position velocity local updates" is not clear what it means. The comments need to be proper sentences.

Reply:
resolved